data/cpplinter.lua

```diff
@@ -2182,7 +2182,6 @@ configKeys = {
 	FRAG_TIME = 21,
 	WHITE_SKULL_TIME = 22,
 	GAME_PORT = 23,
-	LOGIN_PORT = 24,
```

Comment:
You need to fix all of the following enum values

Reply:
Ready! for a moment, I thought it was c++